libraries/src/Component/ComponentHelper.php

```diff
@@ -252,37 +256,37 @@ public static function filterText($text)
 					$filter->blockedAttributes = $customListAttributes;
 				}
 			}
-			// Blacklists take second precedence.
-			elseif ($blackList)
+			// DisallowList take second precedence.
+			elseif ($disallowList)
 			{
-				// Remove the whitelisted tags and attributes from the black-list.
-				$blackListTags       = array_diff($blackListTags, $whiteListTags);
-				$blackListAttributes = array_diff($blackListAttributes, $whiteListAttributes);
+				// Remove the allowed tags and attributes from the disallowList.
```

Comment:
```suggestion
				// Remove the allowed tags and attributes from the Disallow List.
```
Formalise the wording to show as separate words.